### ISSUE TITLE
Should I stop training, change configuration and anchor boxes?

### ISSUE THREAD

**Issue by earcz** [AlexeyAB/darknet#5953]
I am training Yolov3 with BDD100k on 4 GPUs which its configuration as bellow. It seems it will not get better mAP until the training finishes. I am thinking of stopping the training and increase input resolution and maybe use Gaussian-Yolov3. Loss curvature is below.

![multi_gpu](https://user-images.githubusercontent.com/34414342/84575787-9397ac80-adb8-11ea-866d-b2763b9577aa.png)

```
[net]
#Testing
#batch=64
#subdivisions=1

#Training
batch=64
subdivisions=16
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.00025
burn_in=4000
max_batches=320000
policy=steps
steps=256000,288000
scales=.1,.1
```

Also, to experience how anchor box size affects the accuracy, I was planning to deploy another training by calculating anchor boxes size of BDD100k. To do that, I have compared the output of 
**gen_anchors.py** in this repo, [YOLOv3-Calculate-Anchor-Boxes repo for BDD100k](https://github.com/decanbay/YOLOv3-Calculate-Anchor-Boxes) and [the sizes of anchor boxes in cfg file from official Gaussian_YOLOv3 repo](https://github.com/jwchoi384/Gaussian_YOLOv3/blob/18a0a7d255e28103bbfa6cbc709f0e2507c2b420/cfg/Gaussian_yolov3_BDD.cfg#L610). As a result, I obtained really different values from each repo. So, I am not sure which one to use as new anchor box sizes.

**1. Calculation result from gen_anchors.py in this repo:**
```
#for 608x608
anchors = 0.22,0.41, 0.33,1.14, 0.51,0.63, 0.64,2.50, 0.92,1.18, 1.69,2.06, 2.58,3.86, 4.39,5.88, 6.39,10.02
```

**2. Default anchor box sizes of BDD100k in cfg file of original Gaussian_Yolov3 repo:**

```
#for 512x512
anchors = 7,10, 14,24, 27,43, 32,97, 57,64, 92,109, 73,175, 141,178, 144,291

```

**3. Calculation result from YOLOv3-Calculate-Anchor-Boxes repo:**
```
#for 608x608
anchors = 9,16, 21,39, 36,56, 45,92, 78,135, 110,146, 122,122, 185,281, 244,374

#for 512x512
anchors = 8,14, 17,33, 30,47, 38,77, 66,114, 93,123, 102,186, 155,237, 205,314

```

**Comment by AlexeyAB** [comment]
What resolution do you want to use?

What anchors do you get by:
darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 608 -height 608 -show

**Comment by earcz** [comment]
Thanks for prompt reply @AlexeyAB 

I want to use 608x608 resolution.

> darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 608 -height 608 -show

I did not use that command literally. Instead, I used the following:

`python3 gen_anchors.py -filelist train_bdd.txt -output_dir anchors/ -num_clusters 9`

and the output is as:

`anchors = 0.22,0.41, 0.33,1.14, 0.51,0.63, 0.64,2.50, 0.92,1.18, 1.69,2.06, 2.58,3.86, 4.39,5.88, 6.39,10.02`

**Comment by 1027663760** [comment]
Use this command to calculate anchors：

darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 608 -height 608 -show

**Comment by Sankalp-Panghal** [comment]
> Use this command to calculate anchors：
> 
> darknet.exe detector calc_anchors data/obj.data -num_of_clusters 9 -width 608 -height 608 -show

Hey!
1 .Actually, I dont get it whagt do we mean by asking yolo to calculate anchors? So what does that command do?
2. Also Can anyone explain what are these values of anchor and mask in cfg files?
[yolo]
mask = 6,7,8
anchors = 12, 16, 19, 36, 40, 28, 36, 75, 76, 55, 72, 146, 142, 110, 192, 243, 459, 401
classes=1
num=9
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
scale_x_y = 1.05
iou_thresh=0.213
cls_normalizer=1.0
iou_normalizer=0.07
iou_loss=ciou
nms_kind=greedynms
beta_nms=0.6
